manager/integration/tests/test_scheduling.py

```diff
@@ -1754,7 +1758,37 @@ def test_allow_empty_node_selector_volume_setting(): # NOQA
     - Wait for a while for controller to resync the volume,
       all replicas can be scheduled to the nodes
     """
-    pass
+    # Setup
+    node_tag = ["AVAIL"]
+    for node in client.list_node():
+        set_node_tags(client, node, tags=node_tag, retry=True)
+        wait_for_node_tag_update(client, node.name, node_tag)
+
+    update_setting(client, SETTING_ALLOW_EMPTY_NODE_SELECTOR_VOLUME, "false")
+
+    # Check volume can not be scehduled
+    client.create_volume(name=volume_name, size=SIZE)
+    volume = wait_for_volume_detached(client, volume_name)
+
+    volume = client.by_id_volume(volume.name)
+    volume = wait_for_volume_condition_scheduled(client, volume_name,
+                                                 "status",
+                                                 CONDITION_STATUS_FALSE)
+
+    # Rremove tag from 1 node and set setting allow-empty-node-selector-volume
+    # to true
+    node = client.by_id_node(get_self_host_id())
```

Comment:
@chriscchien Do you know why we have to remove tag from a node?

cc @ChanYiLin

Reply:
To verify that when `allow-empty-node-selector-volume = true`, no matter node have tags or not,  all replicas can be scheduled on those nodes.

cc @ChanYiLin